Gem/Code/Source/Components/NetworkWeaponsComponent.cpp

```diff
@@ -335,6 +321,21 @@ namespace MultiplayerSample
 
         weaponInput->m_draw = m_weaponDrawn;
         weaponInput->m_firing = m_weaponFiring;
+
+        uint32_t weaponIndexInt = 0;
+        if (weaponInput->m_firing.GetBit(weaponIndexInt))
+        {
+            // Temp hack for weapon firing due to late ebus binding in 1.14
```

Comment:
Unsure what comment means, what is 1.14 in this context? LY 1.14? In which case is this a hack? Is it better to add something like "Add compensation for weapon firing.."

Reply:
Honestly I'm unsure either, this block of code was moved and I lack the context. I'll see if I can find out.

Reply:
As far as I can tell, this comment isn't relevant anymore so I've removed it.